src/test/java/com/icthh/xm/tmf/ms/communication/lep/XmCommunicationMsLepProcessingApplicationListenerTest.java

```diff
@@ -38,29 +22,25 @@ public class XmCommunicationMsLepProcessingApplicationListenerTest {
     private JavaMailSender javaMailSender;
 
     @Autowired
-    private XmCommunicationMsLepProcessingApplicationListener listener;
+    private XmCommunicationMsLepContextFactory contextFactory;
 
     @Test
     @SuppressWarnings("unchecked")
     public void testBindExecutionContext() {
 
-        ScopedContext context = new DefaultScopedContext("scope");
-        listener.bindExecutionContext(context);
+        LepContext context = (LepContext) contextFactory.buildLepContext(null);
 
-        assertEquals(4, context.getValues().size());
-        assertNotNull(context.getValue(BINDING_KEY_COMMONS, CommonsExecutor.class));
-        assertNotNull(context.getValue(BINDING_KEY_SERVICES, Map.class));
```

Comment:
this check must be changed from map to class

Reply:
I added a second test